tests/virtualization/xen/hotplugging.pm

```diff
@@ -68,6 +104,7 @@ sub run_test {
     # TODO:
     record_info "Memory", "Changing the amount of memory available";
     foreach my $guest (keys %xen::guests) {
+        last if (get_var('VIRT_AUTOTEST') && ($sles_running_version lt '12' or ($sles_running_version eq '12' and $sles_running_sp lt '3')));
```

Comment:
Because we do not define "maximum memory" for virtual machine on old sles, so memory hotplugging may cause problem. Additionally, memory handling or ballooning does not seem to work perfect on old sles either. So we skip this test on old sles.

Reply:
Can you maybe put this as a comment over there?

Reply:
Comment added by record_info. Here is http://10.67.133.40/tests/104